### PR TITLE
Move CSS to external files for cheat sheet IA's. 

### PR DESCRIPTION
**Pull request by MrChrisW**
Cleaning things up. 

+  create share/goodie/crontab_cheat_sheet/crontab_cheat_sheet.css
+  create share/goodie/gimp_cheat_sheet/gimp_cheat_sheet.css
+  create share/goodie/tmux_cheat_sheet/tmux_cheat_sheet.css

### CODE REVIEW

File: share/goodie/crontab_cheat_sheet/crontab_cheat_sheet.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+    .zci--answer .crontab-container {
```
Comment (jagtalon) [inline share/goodie/crontab_cheat_sheet/crontab_cheat_sheet.css]: @MrChrisW could you remove the indentation?

File: share/goodie/crontab_cheat_sheet/crontab_cheat_sheet.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    .zci--answer .crontab-container th {
+        font-weight: bold;
+    }
+    .zci--answer .crontab-container td {
+        vertical-align: top;
+    }
+    .zci--answer .crontab-container dd {
+        margin-bottom: 1ex;
+    }
+    @media(max-width: 704px) {
```
Comment (mintsoft) [inline share/goodie/crontab_cheat_sheet/crontab_cheat_sheet.css]: Personally I tend to put media queries at the top; feel free to ignore me if you disagree however!

**Comment by MrChrisW** [comment]
@jagtalon Done.
@mintsoft Sounds reasonable to me :+1:

**Comment by jagtalon** [comment]
@MrChrisW nice! Please let other people merge your PR next time, though.

**Comment by MrChrisW** [comment]
@jagtalon Thanks for letting me know. 
Shortly after gaining repo permissions, I asked a few of the other devs what the *policy/etiquette* surrounding the merging of PR's was. Unfortunately I didn't receive a firm answer on when it's appropriate to merge/not appropriate. Please feel free to let me know what the standard process for my own PR's or other PR's is? Thanks

**Comment by mintsoft** [comment]
@MrChrisW At the moment I use a "one line" rule. 

If it's a single line change I'll merge it myself after a day or two if no-one else has and there're no objections. Otherwise I wait for someone else to do it. 

@jagtalon @moollaza Any alternative ideas?

**Comment by moollaza** [comment]
@mintsoft @MrChrisW 

In general I try not to merge my own open-source PR's for Spice/Goodie unless I get the reviewer's permission. Even for trivial fixes its best to have someone take a look at it. 

Similarly we aim to never push (especially force push) commits to master. Even if it's a hotfix, we make a branch and generally have someone else merge. In rare cases, we'll self merge, but the goal is to minimize self-merges so that we know at least one other person has looked at, and approved any given changes.

**Comment by MrChrisW** [comment]
@mintsoft @moollaza Thanks for the explanations. Definitely agree, code review is important, I'll follow your advise! :+1: 

Might be a good idea to get this down in writing or even in the documentation for new contributors/commleaders that gain merging permissions. Just so things are clear, could even be included in the welcome email from @zekiel 

Again, thanks for taking the time to explain these things :smile:

**Comment by zekiel** [comment]
@MrChrisW  good idea. will add asap

**Pull request merged**